training-node/test/users.js

```diff
@@ -205,24 +204,22 @@ describe('users', () => {
   });
   describe('/users GET', () => {
     it('should success search ', done =>
-      exports.successUserCreate().then(res =>
-        exports.successUserAuth().then(response => {
-          const token = response.headers[tokenManager.HEADER_NAME];
+      exports.successCommonAuth().then(response => {
+        const token = response.headers[tokenManager.HEADER_NAME];
 
-          return chai
-            .request(server)
-            .get('/users?offset=0&limit=2')
-            .set(tokenManager.HEADER_NAME, token)
-            .then(result => {
-              dictum.chai(result);
-              result.should.have.status(200);
-              result.body.should.have.property('results');
-              result.body.results.should.have.length(2);
-              result.body.should.have.property('total');
-              done();
-            });
-        })
-      ));
+        return chai
```

Comment:
for the pagination, you can create multiple users (like 100) and check that for some pages its ok